Gopkg.lock

```diff
@@ -165,6 +173,7 @@
     "github.com/shirou/gopsutil/process",
     "github.com/sirupsen/logrus",
     "github.com/stretchr/testify/assert",
+    "github.com/troian/dmidecode",
```

Comment:
any reason it is not included as an internal pkg?

Reply:
moved to github.com/cloudradar-monitoring/dmidecode as discussed in slack